README.adoc

```diff
@@ -41,9 +41,24 @@ Select the tenant. On Hawkular servers, use `hawkular`.
 
 Openshift-Metrics users must provide an authentication token.
 
+== Using Grafana Templating (variables)
```

Comment:
Do you think it would be possible to take a screenshot, add it to the repo and link it here? It would help to understand the following section.